--- a/pymc_marketing/model_builder.py
+++ b/pymc_marketing/model_builder.py
@@ -837,7 +837,8 @@
             var_names = [var.name for var in model.free_RVs]
             idata = pm.sample(var_names=var_names, **kwargs)
             idata["/posterior"] = pm.compute_deterministics(
-                idata["/posterior"], merge_dataset=True
+                idata["/posterior"], merge_dataset=True,
+                progressbar=kwargs.get("progressbar", True),
             )
         return idata
 
